benchmark-sets/comparison/abseil-cpp.yaml

```diff
@@ -0,0 +1,65 @@
+"functions":
+- "name": "bool absl::str_format_internal::FormatArgImpl::Dispatch<long long>(absl::str_format_internal::FormatArgImpl::Data, absl::str_format_internal::FormatConversionSpecImpl, void*)"
```

Comment:
What's the difference between name and signature now? Should we just make this all simpler and have just a single field?

Reply:
Yes, I created #92 for this.

I prefer to replace all function name usages with signatures in another PR, given this mainly focuses on adding more details to benchmark YAML.
Would this be reasonable or would you prefer me to fix that in this PR?

Reply:
SGTM. please add more text/details to the bug to better explain the context :)

Reply:
Thanks, done!